src/DestinationCard/index.js

```diff
@@ -0,0 +1,293 @@
+  opacity: 0;
+  position: relative;
+  top: "-50%";
+  margin-bottom: ${({ theme }) => theme.orbit.spaceXXSmall};
+`;
+
+Shown.defaultProps = {
+  theme: defaultTokens,
+};
+
+const ArrowUp = styled(ArrowUpIcon)`
+  transform: rotate(90deg);
+  margin-left: ${({ theme }) => theme.orbit.spaceXXSmall};
+`;
+
+ArrowUp.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledDestinationCardHiddenContent = styled.div`
+  padding: ${({ theme }) => theme.orbit.spaceSmall} 0 0;
+  border-top: 1px solid rgba(255, 255, 255, 0.3);
+`;
+
+StyledDestinationCardHiddenContent.defaultProps = {
+  theme: defaultTokens,
+};
+
+const FlightDirect = styled(FlightDirectIcon)`
+  transform: ${({ isReturn }) => isReturn && "rotate(180deg)"};
+`;
+
+const StyledDestination = styled.div`
+  position: relative;
+  transition: top ${({ theme }) => theme.orbit.durationFast} ease-in-out;
+  top: 16px;
+`;
+
+StyledDestination.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledDestinationCard = styled(({ height, imageURL, theme, ...props }) => <div {...props} />)`
+  width: 100%;
+  height: ${({ height }) => (height ? `${height}px` : "100%")};
+  display: flex;
+  align-items: flex-end;
+  position: relative;
+  box-sizing: border-box;
+  border-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  overflow: hidden;
+  cursor: pointer;
+
+  &:hover,
+  &:focus {
+    ${StyledOverlay} {
+      opacity: 0;
+    }
+    ${StyledDestination} {
+      top: 0px;
+    }
+    ${StyledOverlayHover} {
+      opacity: 1;
+    }
+    ${StyledDestinationCardContent} {
+      bottom: 0;
+    }
+    ${Shown} {
+      opacity: 1;
+      top: 0;
+    }
+  }
+`;
+StyledDestinationCard.defaultProps = {
+  theme: defaultTokens,
+};
+
+const SmallHeading = ({ children }: SmallHeadingType) => (
+  <Text type="white" element="div" size="small" weight="bold">
+    {children}
+  </Text>
+);
+
+class DestinationCard extends React.PureComponent<Props, State> {
+  state = {
+    hiddenContentHeight: 0,
+  };
+
+  componentDidMount() {
+    this.setHeight();
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps !== this.props) {
+      this.setHeight();
+    }
+  }
+
+  setHeight = () => {
+    const { hiddenContent } = this;
+
+    if (hiddenContent.current) {
+      this.setState({
+        hiddenContentHeight: hiddenContent.current.clientHeight,
+      });
+    }
+  };
+
+  hiddenContent: { current: any | HTMLDivElement } = React.createRef();
+
+  render() {
+    const {
+      dataTest,
+      departureCity,
+      destinationCity,
+      destinationCountry,
+      image,
+      price,
+      timeOfStay,
+      outbound,
+      inbound,
+      height = 300,
+      onClick,
+    } = this.props;
+    const { hiddenContentHeight } = this.state;
+    return (
+      <StyledDestinationCard
+        data-test={dataTest}
+        onClick={onClick}
+        height={height >= 175 ? height : SMALLEST_HEIGHT}
```

Comment:
The `SMALLEST_HEIGHT` has value of `200`. So I would use condition: `height >= SMALLEST_HEIGHT ? height : SMALLEST_HEIGHT`.
And use either `175` or `200`, up to you.